TeamCode/src/main/java/org/firstinspires/ftc/teamcode/autonomous/DemoAuto.kt

```diff
@@ -0,0 +1,33 @@
+package org.firstinspires.ftc.teamcode.opmode.autonomous
+
+import com.qualcomm.robotcore.eventloop.opmode.Autonomous
+import com.qualcomm.robotcore.eventloop.opmode.Disabled
+import com.qualcomm.robotcore.eventloop.opmode.LinearOpMode
+import com.qualcomm.robotcore.util.ElapsedTime
+import org.firstinspires.ftc.teamcode.api.DemoAPI
+import org.firstinspires.ftc.teamcode.api.linear.DemoLinearAPI
+
+@Autonomous(name = "Demo Auto")
```

Comment:
```suggestion
@Autonomous(name = "Demo Auto", group = "Demo")
```

I like to add a group to these opmodes so that they are all grouped together in the UI on the driver hub.